_sass/members.scss

```diff
@@ -1,9 +1,4 @@
-* {
-	margin: 0;
-	padding: 0;
-	-webkit-box-sizing: border-box;
-	box-sizing: border-box;
-}
+
 h1 {
 	font-size: 2.5rem;
```

Comment:
H1 tag seems to have a global css value is it intentional?

Reply:
Actually it is the code of suhani

Reply:
ohh I see @suhanichawla will u be changing it?

Reply:
I can change it at the end, just need to preceed it with .wrapper class